core/drivers/crypto/caam/caam_jr.c

```diff
@@ -602,9 +603,9 @@ enum caam_status caam_jr_init(struct caam_jrcfg *jrcfg)
 	jr_privdata->it_handler.handler = caam_jr_irqhandler;
 	jr_privdata->it_handler.data = jr_privdata;
 
-#ifdef CFG_NXP_CAAM_RUNTIME_JR
-	itr_add(&jr_privdata->it_handler);
-#endif
+	if (IS_ENABLED(CFG_NXP_CAAM_RUNTIME_JR))
```

Comment:
Those changes are not needed in the case of adding the HUK generation with the CAAM Blob operation

Reply:
This change is needed to resolve a compile error when `CFG_NXP_CAAM_RUNTIME_JR=n`.